kevm-pyk/src/kevm_pyk/kproj/evm-semantics/evm.md

```diff
@@ -1039,14 +1039,14 @@ The `JUMP*` family of operations affect the current program counter.
 
     syntax UnStackOp ::= "JUMP"
  // ---------------------------
-    rule <k> JUMP DEST => #endBasicBlock... </k>
+    rule <k> JUMP DEST => #endBasicBlock ... </k>
          <pc> _ => DEST </pc>
          <jumpDests> DESTS </jumpDests>
-      requires DEST in DESTS
+      requires DESTS[DEST] ==Int 1
```

Comment:
We need a check that `DEST` is within bounds of the buffer.

Reply:
Right, great catch!